Classes/ViewHelpers/Content/GetViewHelper.php

```diff
@@ -127,9 +129,10 @@ public static function renderStatic(
         }
         $templateVariableContainer = $renderingContext->getVariableProvider();
         $record = (array) $renderingContext->getViewHelperVariableContainer()->get(FormViewHelper::class, 'record');
-
-        if ($GLOBALS['BE_USER']->workspace) {
-            $placeholder = BackendUtility::getMovePlaceholder('tt_content', $record['uid']);
+        
+        $workspaceId = GeneralUtility::makeInstance(Context::class)->getPropertyFromAspect('workspace', 'id');
```

Comment:
This breaks backwards compatibility with TYPO3 versions that don't have the Context class. It is not safe to assume that it exists. We would need a check for `class_exists` and a fallback to use the old method in case it doesn't exist.

Reply:
The Fallback was implemented